<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>rubric_form</class>
 <widget class="QDialog" name="rubric_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>529</width>
    <height>278</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add/Edit rubric</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="rubric_name_layout">
     <item>
      <widget class="QLabel" name="lRubricName">
       <property name="text">
        <string>Rubric name :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignJustify|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lERubricName"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="search_template_layout">
     <item>
      <widget class="QLabel" name="lSearchTemplate">
       <property name="text">
        <string>Search template :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignJustify|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lESearchTemplate">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tbSearchTemplate">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="category_layout">
     <item>
      <widget class="QLabel" name="lCategory">
       <property name="text">
        <string>Source category :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignJustify|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lECategory">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tbCategory">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="io_layout">
     <item>
      <widget class="QLabel" name="lIO">
       <property name="text">
        <string>Source qualifier :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignJustify|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lEIO">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tbIO">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="rubric_desc_layout">
     <item>
      <widget class="QLabel" name="lDescription">
       <property name="text">
        <string>Rubric description :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignJustify|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lERubricDescription"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="icon_layout">
     <item>
      <widget class="QLabel" name="lRubricIcon">
       <property name="text">
        <string>Rubric icon :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignJustify|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lRubricIconImage">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tbRubricIcon">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="gbAccessRules">
     <property name="title">
      <string>Access rules</string>
     </property>
     <layout class="QGridLayout" name="gridLayout"/>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>398</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbOk">
       <property name="text">
        <string>&amp;OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbCancel">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <zorder>gbAccessRules</zorder>
  <zorder>verticalSpacer</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
